### PR TITLE
added recipe for `bids-validator`

### PR DESCRIPTION
**Pull request by sarthakpati**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by effigies** [comment]
I agree to be a maintainer

**Comment by sappelhoff** [comment]
I agree to be a maintainer.

**Comment by carterbox** [comment]
@conda-forge/help-python @conda-forge/staged-recipes this recipe is ready for review.

**Comment by ocefpaf** [comment]
> @conda-forge/help-python @conda-forge/staged-recipes this recipe is ready for review.

We need @rwblair acknowledgement to become a maintainer.

File: recipes/bids-validator/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  home: https://github.com/bids-standard/bids-validator
+  summary: Validator for the Brain Imaging Data Structure
+  license: MIT
+  license_file:
+    - LICENSE
+
+extra:
+  recipe-maintainers:
+    - effigies
+    - rwblair
```
Comment (effigies) [inline recipes/bids-validator/meta.yaml:42]: Let's remove Ross for now. I work with him and am happy to add him later if he wants.

```suggestion
```

Comment (ocefpaf) [reply recipes/bids-validator/meta.yaml:42]: Just some background on why we ask this. There will be some automated emails from GitHub adding folks to the feedstock. Some folks get really mad at us for this so we need to be sure they know what they are agreeing to. Note that, in the feedstock, you can add them without this step. B/c there it is your responsibility to communicate with the maintainers, not the conda-forge core members.

**Comment by BastianZim** [comment]
There was an error in the bot but it has been resolved. Will take some time, though until the backlog is solved.

**Pull request merged**